helm/druid/templates/configmap.yaml

```
@@ -31,7 +31,7 @@ data:
 {{ toYaml .Values.configVars | indent 2 }}
 {{- if .Values.zookeeper.enabled }}
   druid_zk_service_host: {{ .Release.Name }}-zookeeper-headless:2181
-{{- else }}
+{{- else .Values.zkHosts }}
```

Pretty sure this should be:
```suggestion
{{- else if .Values.zkHosts }}
```

---

Ugh, yes I thought I had that.  Thank you for catching.  Will fix now.

---

Fixed.